.github/workflows/release-copy-github-io-to-sourceforge.yml

```diff
@@ -2,8 +2,8 @@
 # GitHub Action to Copy Site to SourceForge.
 #
 #############################################################################
-name: "R: Copy Site to SourceForge"
-run-name: "R: Copy Site to SourceForge ${{ inputs.version }}"
+name: "R: Copy github.io to SourceForge"
+run-name: "R: Copy github.io to SourceForge ${{ inputs.version }}"
```

Comment:
`github` should be `GitHub` such as https://docs.github.com/en/get-started/writing-on-github

Reply:
Ok to change. 

Or change SourceForge to sourceforge.org

Reply:
Before I push, is `R: Copy github.io to sourceforge.org` okay?

Reply:
I dunno, are we going to change `release-copy-github-io-to-sourceforge.yml` to `release-copy-github-io-to-sourceforge-org.yml`. It seems excessively specific.

I am fine with whatever @romani decides, but I think it should just be GitHub and Sourceforge` as they are the names of the sites and the actual domains shouldn't matter for just simple naming.

Reply:
Done.